lxd/daemon.go

```diff
@@ -528,6 +540,10 @@ func (d *Daemon) Init() error {
 	/* Detect user namespaces */
 	runningInUserns = shared.RunningInUserNS()
 
+	if !haveMacAdmin() {
+		shared.Log.Warn("Per-container AppArmor profiles are disabled because mac_admin capability is missing.")
+		aaEnabled = false
+	}
```

Comment:
Should probably move after the /sys/kernel/security test.

I've been trying to order things from the least resource heavy to the most resource heavy to optimize things a bit (probably not noticeable on a typical lxd host though).